guide/english/java/interfaces/index.md

```diff
@@ -32,7 +32,9 @@ public class Car implements Vehicle {
 
 ![:rocket:](//forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=2 ":rocket:") <a href='https://repl.it/CItd/0' target='_blank' rel='nofollow'>Run Code</a>
 
-Now, there is a **ground rule** : The Class must implement **all** of the methods in the Interface. The methods must have _the exact same_ signature (name, parameters and exceptions) as described in the interface. The class _does not_ need to declare the fields though, only the methods.
+Now, there is a **ground rule** : The Class must implement **all** of the methods in the Interface. The methods must have _the exact same_ signature (name, parameters) as described in the interface. However they do not necessary raise all exceptions declared in the interface, but they cannot raise exceptions that are not declared in the interface. The class _does not_ need to declare the fields though, only the methods.
```

Comment:
> However they do not necessary raise all exceptions declared in the interface, but they cannot raise exceptions that are not declared in the interface. 

This statement is not clear. Please re-phrase it, or remove it entirely to revert to the original description.